### PR TITLE
Adding EV charging related measures

### PR DESCRIPTION
**Pull request by amyeallen1**
Hi Nick and Katherine, David suggested that I submit these two EV charging related measures that I wrote to this repo. Shanti would like to make this work publicly available. @nllong @kflemin Do one of you two review measures for this repo or should I ask someone else? Thanks!

### CODE REVIEW

**Comment by kflemin** [comment]
@amyeallen1, some of your files are missing a blank line (you can see them if you click on your commit). Otherwise as long as you have tests and they pass you should be good to go.  I'd have @nllong review this too.

**Comment by amyeallen1** [comment]
Thanks, Katherine! @kflemin I think the flies that are tagged as missing the blank line were automatically generated from running the test on the example model file, such as the results.json. Should I include the results from the tests in the commit?

**Comment by kflemin** [comment]
@amyeallen1 no, I would not include test results in the commits.  They should be ignored automatically via the .gitignore file.  I'm not sure why they should up here.

**Comment by nllong** [comment]
I second @kflemin in not including the test results. You will have to do a weird work around to remove them from git (for real) since you have included them already. The approach would be to do the following:

* remove the files locally
* run `git add -u` to stage the removed files.
* run `git commit --amend`  . This works because you only have one commit (otherwise I would use interactive rebase)
* then you will need to force push. (this should always be done with caution and only on branches that are yours). `git push --force`

**Comment by amyeallen1** [comment]
Thanks Nick and Katherine! I just followed those instructions, Nick. @nllong @kflemin

File: lib/measures/add_ems_to_control_ev_charging/measure.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <datatype>string</datatype>
+    </attribute>
+    <attribute>
+      <name>Intended Software Tool</name>
+      <value>Parametric Analysis Tool</value>
+      <datatype>string</datatype>
+    </attribute>
+  </attributes>
+  <files>
+    <file>
```
Comment (nllong) [inline lib/measures/add_ems_to_control_ev_charging/measure.xml:55]: Do we need to add all of the load profile files to the XML @kflemin

**Comment by amyeallen1** [comment]
Thanks, Nick and Katherine! @nllong @kflemin

**Pull request merged**